corehq/motech/repeaters/models.py

```diff
@@ -196,17 +196,14 @@ def all_ready(self):
                 .filter(repeat_records_ready_to_send))
 
 
-class RepeaterStub(models.Model):
-    """
-    This model links the SQLRepeatRecords of a Repeater.
-    """
+class SQLRepeater(models.Model):
     domain = models.CharField(max_length=126)
     repeater_id = models.CharField(max_length=36)
     is_paused = models.BooleanField(default=False)
     next_attempt_at = models.DateTimeField(null=True, blank=True)
     last_attempt_at = models.DateTimeField(null=True, blank=True)
 
-    objects = RepeaterStubManager()
+    objects = RepeaterManager()
 
     class Meta:
```

Comment:
Should this include `db_table` as mentioned in the [Couch to SQL models doc](https://github.com/dimagi/commcare-hq/blob/d3b0bc46a5021315586986ae1e2e22eed2aaed92/docs/couch_to_sql_models.rst#pr-1-add-sql-model-and-migration-management-command-write-to-sql)? (search for `db_table`)